src/components/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  filterBy: string;
+  setFilterBy: (param: string) => void;
+  filteredTodos: Todo[];
+};
+
+export const Footer: React.FC<Props> = ({
```

Comment:
when user doesn't have any tasks you should hide filters
like on [referpage](https://mate-academy.github.io/react_todo-app-with-api/)
<img width="1510" alt="image" src="https://user-images.githubusercontent.com/99652389/194323470-7465caf7-b605-4819-8a5e-93308d5111a2.png">

Reply:
also, check the checklist. there you have hint how to name components)

Reply:
I fixed it in next part)